docs/skopeo-copy.1.md

```diff
@@ -30,6 +30,10 @@ If the authorization state is not found there, $HOME/.docker/config.json is chec
 
 **--sign-by=**_key-id_ add a signature using that key ID for an image name corresponding to _destination-image_
 
+**--encryption-key** _Key_ a reference prefixed with the encryption protocol to use. e.g. jwe:/path/to/key.pem or pgp:admin@example.com or pkcs7:/path/to/x509-file
```

Comment:
`e.g.` -> `For instance,`.

Reply:
Which protocols are supported? Are there any other requirements? We should add a sentence (or two) about that.